tests/02_eradiate/02_system/conftest.py

```diff
@@ -1,9 +1,80 @@
 import numpy as np
+import pint
 import pytest
 import xarray as xr
 
 import eradiate
 from eradiate import unit_registry as ureg
+from eradiate.units import symbol
+
+
+def rayleigh_phase_function(mu: np.ndarray) -> pint.Quantity:
+    """Evaluate Rayleigh phase function."""
+    magnitude = 3.0 * (1 + np.square(mu)) / (16 * np.pi)
+    return magnitude / ureg.steradian
+
+
+def to_dataset(
```

Comment:
I think a name reflection this function intent as a data creation component would be more appropriate, like `make_dataset()`. Anyway, since it's not reused anywhere else, I find it a bit confusing to make it a function. If you absolutely want an intermediate name, you can just as well create a variable.